README.md

```diff
@@ -1,5 +1,7 @@
 # Quickstart
 
+[![Open in Orchest](https://github.com/orchest/orchest-examples/raw/main/imgs/open_in_orchest.svg)](https://cloud.orchest.io/?import_url=https://github.com/astrojuanlu/quickstart/)
```

Comment:
I guess you wanted to do `import_url=https://github.com/orchest/quickstart/`? :stuck_out_tongue_winking_eye:

Reply:
🤦🏽‍♂️